zou/app/blueprints/user/resources.py

```diff
@@ -25,7 +24,6 @@ class AssetTasksResource(Resource):
     Return tasks related to given asset for current user.
     """
 
-    @jwt_required()
```

Comment:
Why do you remove authentication from all these routes?

Reply:
It's because it's directly set in all routes of the blueprint : https://github.com/cgwire/zou/pull/771/files#diff-773e47b9349c32eebf93133c5858bfb2383e2f8f29c56d61e2dbecb079a63307R110
This blueprint is blocked for bots